src/tests/electron/common/element-identifiers/automated-checks-view-selectors.ts

```diff
@@ -23,6 +23,8 @@ export const AutomatedChecksViewSelectors = {
         `${nthRuleGroup(position)} [data-automation-id="${cardsRuleIdAutomationId}"]`,
     nthRuleGroupInstances: (position: number) =>
         `${nthRuleGroup(position)} [data-automation-id="${instanceCardAutomationId}"]`,
+
+    settingsButton: 'button[aria-label="settings"]',
```

Comment:
Consider instead creating a data-automation-id for it?

Reply:
done